Makefile

```
@@ -316,6 +341,97 @@ test-integration-watch: ## Run integration tests in watch mode (TEST_ENV=integra
+
+ci-prod-setup: ## Prepare the prod + Chromium environment for prod-side CI tests
+	$(MAKE) start-prod
+	$(MAKE) install-chromium-lhci
+
+ci-test-e2e: ## Run E2E tests assuming ci-prod-setup already started the prod environment
+	$(run-e2e)
+
+ci-test-visual: ## Run visual tests assuming ci-prod-setup already started the prod environment
+	$(run-visual)
+
+ci-test-memory-leak: ## Run Memlab memory leak tests against the dedicated compose stack (assumes prod is running)
```

checkmake (0.3.2)

[warning] Target body for "ci-test-memory-leak" exceeds allowed length of 5 lines (11).

(maxbodylength)